### PR TITLE
Bump version number from release

### PR DESCRIPTION
**Pull request by mklauber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tiddlywiki",
   "preferGlobal": "true",
-  "version": "5.1.16-prerelease",
+  "version": "5.1.16",
```
Comment (pmario) [inline package.json]: 5.1.16 is already released. So it needs to be 5.1.17-prerelease now.

Comment (mklauber) [reply package.json]: The issue is 5.1.16 was released without removing the prerelease flag.  So stable 5.1.16 lists it's version as 5.1.16-prerelease.  This is an attempt to fix that.

Comment (mklauber) [reply package.json]: See https://github.com/Jermolene/TiddlyWiki5/issues/3239

Comment (pmario) [reply package.json]: ahh .. OK. .. my bad

**Comment by Jermolene** [comment]
Thanks @mklauber I'll push a fix shortly

**Pull request closed without merging**